desktop_version/src/Map.cpp

```diff
@@ -2101,9 +2101,6 @@ void mapclass::twoframedelayfix()
 	game.state = 0;
 	game.statedelay = 0;
 	script.load(game.newscript);
-	if (script.running)
-	{
-		script.run();
-		script.dontrunnextframe = true;
-	}
+	script.run();
+	script.dontrunnextframe = true;
```

Comment:
Do we care that this now gets unconditionally set to true?

Reply:
It shouldn't matter. It'll be removed when #535 gets merged anyway.